dashboard/templates/base.html

```diff
@@ -19,6 +19,7 @@
 </head>
 <body>
 {% block navbar %}
+    {% set current_user = get_current_user() %}
```

Comment:
Why set this in template instead of template_context? I think we can use ``'current_user': users.get_current_user()``